### ISSUE TITLE
Exclusions

### ISSUE THREAD

**Issue by gamesaccounta** [UnamSanctam/UnamBinder#56]
How can I change the exclusions UnamBinder makes?

**Comment by UnamSanctam** [comment]
On this line:
https://github.com/UnamSanctam/UnamBinder/blob/b79b6f3eb56bd417e089da35393efa4396f31163/UnamBinder/Forms/Builder.cs#L251

**Comment by gamesaccounta** [comment]
I wish to add System32, AppData and ProgramFiles replacing "$env:UserProfile,$env:SystemDrive" to "$env:System32,$env:ProgramFiles,$env:AppData", but it doesnt work, can you help me?

**Comment by UnamSanctam** [comment]
I don't believe `$env:System32` exists, but you can use `[Environment]::SystemDirectory` instead I think.

**Comment by gamesaccounta** [comment]
How can I do that?

**Comment by UnamSanctam** [comment]
`@([Environment]::SystemDirectory,$env:ProgramFiles,$env:AppData)`

**Comment by gamesaccounta** [comment]
That adds program files x86, i want programfiles

**Comment by gamesaccounta** [comment]
And it adds AppData\Roaming, i want all AppData

**Comment by UnamSanctam** [comment]
>That adds program files x86, i want programfiles

That should not add the x86 Program Files unless you somehow has the environmental variable set as that or if it's due to a WoW64 subsystem, you can try using `$env:ProgramW6432`
<img src="https://github.com/UnamSanctam/UnamBinder/assets/10323655/2d49cf1e-be69-4324-aac8-16f21698fd3e">

You can see what each environmental variable points to by running the `env` command inside a command line.

>And it adds AppData\Roaming, i want all AppData

Yes `AppData` usually refers to `AppData\Roaming`, if you want the base folder then you'd have to traverse back one folder with something like `Join-Path -Path $env:AppData -ChildPath '..'`

**Comment by gamesaccounta** [comment]
Is it well? Join-Path -Path $env:AppData -ChildPath '..' -ExclusionPath @([Environment]::SystemDirectory,$env:ProgramFiles,$env:AppData)

**Comment by UnamSanctam** [comment]
It would be something like this (I also changed the command):
`-ExclusionPath @([Environment]::SystemDirectory,$env:ProgramW6432,(Split-Path $env:APPDATA))`

**Comment by gamesaccounta** [comment]
Thanks it works sir, how can I donate you for this huge help?

**Comment by UnamSanctam** [comment]
Thank you, I have (crypto) donation links on this repository main page at the bottom, but only give something if you have enough to give away.
